### PR TITLE
Homework Podstawy C++ #2

### PR DESCRIPTION
**Pull request by dawidjozefczak**

### CODE REVIEW

File: module2/homework/addEven/addEven.hpp [hunk 1]
```diff
@@ -1,8 +1,12 @@
 #pragma once
 #include <vector>
 
-int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+int addEven(const std::vector<int>& numbers){
```
Comment (m-kop) [inline module2/homework/addEven/addEven.hpp:4]: missing space

File: module2/homework/addEven/addEven.hpp [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+    int result = 0;
+    for(const auto& element : numbers){
```
Comment (m-kop) [inline module2/homework/addEven/addEven.hpp:6]: ditto

File: module2/homework/addEven/addEven.hpp [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
+        if(element % 2 == 0){
```
Comment (m-kop) [inline module2/homework/addEven/addEven.hpp:7]: ditto

File: module2/homework/generateSequence/generateSequence.hpp [hunk 4]
```diff
@@ -1,7 +1,14 @@
 #pragma once
 #include <vector>
 
-std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+std::vector<int> generateSequence(int count, int step){
```
Comment (m-kop) [inline module2/homework/generateSequence/generateSequence.hpp:4]: missing space

File: module2/homework/nwdNww/nwdNww.hpp [hunk 5]
```diff
@@ -1,11 +1,19 @@
...
+        lhs = temp;
+    }
+    return abs(lhs);   
 }
 
-int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+int NWW(int lhs, int rhs){
+    if(lhs == 0 && rhs == 0){
```
Comment (m-kop) [inline module2/homework/nwdNww/nwdNww.hpp:14]: or

File: module2/homework/generateSequence/generateSequence.hpp [hunk 6]
```diff
@@ -1,7 +1,14 @@
 #pragma once
 #include <vector>
-
-std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+#include <iostream>
+std::vector<int> generateSequence(int count, int step){
```
Comment (kyrtaq) [inline module2/homework/generateSequence/generateSequence.hpp:4]: you can add consts

**Comment by ziobron** [comment]
+4 XP addEven
+6 XP NwwNwd
+5 XP maxOfVector
+5 XP generateSequence
+8 XP delivery before 31.05

{ "requirement_ids": [1119, 1631, 1119, 1633, 1119, 1634, 1119, 1635], "points": [2, 4, 2, 6, 2, 5, 2, 5] }

**Pull request closed without merging**